drivers/soundwire/cadence_master.c

```diff
@@ -368,6 +372,87 @@ static int cdns_hw_reset(void *data, u64 value)
 
 DEFINE_DEBUGFS_ATTRIBUTE(cdns_hw_reset_fops, NULL, cdns_hw_reset, "%llu\n");
 
+static int cdns_parity_error_injection(void *data, u64 value)
+{
+	struct sdw_cdns *cdns = data;
+	struct sdw_slave *slave;
+	struct sdw_bus *bus;
+	int ret;
+
+	if (value != 1)
+		return -EINVAL;
+
+	bus = &cdns->bus;
+
+	/*
+	 * wake up master. If this results in a bus reset, the Slave
+	 * devices will re-attach and be re-enumerated.
+	 *
```

Comment:
nitpick: superfluous line

Reply:
And can we go on using master & slave?

Reply:
> nitpick: superfluous line

fixed.

Reply:
> And can we go on using master & slave?

I am referring to normative terms of a 2015 specification, and use a capital letter to show I am referring to a concept described in that spec.